css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -37,28 +37,49 @@ main {
 
 .box--top-left {
   background-color: #aed6f1;
+  position: absolute;
+  top: 0;
+  left: 0;
```

Comment:
correct. other option would have been to not add the position modifier at all since 0, 0 is the default.